packages/mirage-conduit/mirage-conduit.1.0.0/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/mirage/ocaml-conduit/releases/download/v1.0.0/conduit-1.0.0.tbz"
+checksum: "073ea06f539e0106b1d8a58eb3dd3a68"
```

Comment:
That checksum is wrong, it shoud be b885ef5b3a503c9eda1b5c18bec859b6 like for other archives.